pontoon/base/forms.py

```diff
@@ -163,6 +163,15 @@ class Meta:
         fields = ('first_name',)
 
 
+class UserCustomHomepageForm(forms.ModelForm):
+    """
+    Form is responsible for saving custom home page.
+    """
+    class Meta:
+        model = UserProfile
+        fields = ('custom_homepage',)
+
+
 class UserLocalesSettings(forms.ModelForm):
     """
```

Comment:
Not directly related to this PR, but I'd change the names of these forms:
`UserProfileForm` -> `UserFirstNameForm`
`UserCustomHomepageForm` -> this one is OK
`UserLocalesSettings` -> `UserLocalesOrder`

UserProfileForm also misses a comment: *Form is responsible for saving user's name.*